examples/tf/ppo_pendulum.py

```diff
@@ -71,6 +71,7 @@ def tf_ppo_pendulum(ctxt=None, seed=1):
             entropy_method='max',
             policy_ent_coeff=0.02,
             center_adv=False,
+            fit_baseline_first=True,
```

Comment:
this is hard to understand if i don't know the internal of this algorithm.

maybe you can make it a string arg named 'fit_baselin' instead, with options `'before'` and `'after'`

later we might make an in-graph baseline (that is, the baseline and policy are in the same model), in which case we could introduce the option `'joint'` or `'during'` or something.